index.html

```diff
@@ -0,0 +1,136 @@
+            <p>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda, expertos en diversión y un menú deliciosamente divertido. 
+            <br><br>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda. expertos en diversión y un menú deliciosamente divertido. 
+            </p>    
+            <div><button type="button" name="button">RESERVA</button></div>            
+        </div>
+        <div><img src="images/people.png" alt=""></div>          
+    </div>
+    <div class="segment-three">
+        <div class="content">
+            <h2>PROMOCIONES</h2>
+            <div class="wrapper">
+                <div class="box a"><img src="images/promo1.png" alt=""></div>
+                <div class="box b"><img src="images/promo2.jpg" alt=""></div>
+                <div class="box c"><img src="images/promo_nada.jpg" alt=""></div>
+                <div class="box d"><img src="images/promo3.jpg" alt=""></div>
+                <div class="box e"><img src="images/promo_nada.jpg" alt=""></div>
+                <div class="box f"><img src="images/promo4.jpg" alt=""></div>
+                <div class="box g"><img src="images/promo_nada.jpg" alt=""></div>
+              </div>
+        </div>
+    </div>
+    <div class="segment-four">
+        <div class="content">
+            <div><h2>MENÚ</h2></div>
+            <ul class="menu-top">                    
+                <li><a href="#">BEBIDAS</a></li>
+                <li><a href="#">ENSALADAS</a></li>
+                <li><a href="#">POSTRES</a></li>
+                <li><a href="#">ENTRADAS</a></li>
+                <li><a href="#">PIZZAS</a></li>
+                <li><a href="#">PEPITOS</a></li>
+                <li><a href="#">HAMBURGUESAS</a></li>
+                <li><a href="#">CORTES</a></li>
+            </ul> 
+        </div>
+    </div>
+    <div class="segment-five">
+        <div class="content">
+            <div><h2>¡HAZ UNA RESERVACIÓN!</h2></div>
+            <span>¿Quieres reservar o hacernos alguna pregunta? Envíanos tus datos y te contactaremos a la brevedad :)</span>
+            <span>o llámanos al</span>
+            <span><i class="fas fa-phone"></i>
+            3629 4947
+            </span>
+            <input type="Nombre" placeholder="Nombre*" name="name" id="">
+            <input type="email" placeholder="Email*" name="email" id="">
+            <input type="Nombre" placeholder="Celular*" name="cel" id="">
+            <input type="Nombre" placeholder="Fecha*" name="name" id="">
+            <input type="Nombre" placeholder="Hora*" name="name" id="">
+            <input type="Nombre" placeholder="Comentarios*" name="name" id="">
+            <button type="button" name="button">QUIERO MI RESERVACIÓN</button>                     
+            </ul> 
+        </div>
+    </div>
+    <div class="footer-one">
+        <div class="content">
```

Comment:
For the contact information will be great to use `address` tag
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address